connectors/sources/zoom.py

```diff
@@ -341,6 +341,12 @@ def get_default_configuration(cls):
                 "sensitive": True,
                 "type": "str",
             },
+            "recording_age": {
+                "label": "Recording Age Limit (Months)",
+                "order": 5,
```

Comment:
This is pedantic, but let's put this below `fetch_past_meeting_details` so that the `order` matches the order they are in this list.